scripts/test_eval_attribution.py

```
@@ -0,0 +1,79 @@
+import unittest
+
+from scripts.eval_attribution import UciEngine, parse_exact_score
+
+
+class ParseExactScoreTest(unittest.TestCase):
+    def test_new_game_waits_for_engine_reset(self):
+        engine = object.__new__(UciEngine)
+        events = []
+
+        def send(command):
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `send`

Add return type annotation: `None`

(ANN202)